tests/storage/storage.go

```diff
@@ -138,17 +138,17 @@ var _ = SIGDescribe("Storage", func() {
 				tests.RemoveSCSIDisk(nodeName, address)
 			})
 
-			It("should pause VMI with error disk on IO error", func() {
+			It("[QUARANTINE] should pause VMI on IO error", func() {
```

Comment:
Do we want this?

Reply:
Well, it is still quarantined right? Until we prove it is stable.

Or do you mean you want to see a few runs without the label, and then add label again?

Reply:
ok, It is a leftover from a rebase, let me remove this

Reply:
removed